src/libraries/Native/Unix/System.Security.Cryptography.Native.Android/pal_ecdsa.c

```diff
@@ -3,6 +3,7 @@
 
 #include "pal_ecdsa.h"
 #include "pal_bignum.h"
+#include "pal_signature.h"
 #include "pal_utilities.h"
 
 static jobject AndroidCryptoNative_GetEsDsaSignatureObject(JNIEnv* env)
```

Comment:
Existing typo:

```suggestion
static jobject AndroidCryptoNative_GetEcDsaSignatureObject(JNIEnv* env)
```

Or, since it's a static function:

```suggestion
static jobject GetEcDsaSignatureObject(JNIEnv* env)
```